relaxing cabal constraints

soenkehahn opened the pull request:
I've relaxed some of the cabal constraints and added a travis file. Since I'm listed as the sole maintainer I took the liberty to upload a new version to hackage.

Would you mind transferring the repo to the github organization `nikki-and-the-robots`? Then I would take care of merging this (and updating the cabal file to reflect the change).

soenkehahn committed:
add travis file

soenkehahn committed:
tweaked cabal version constraints

soenkehahn committed:
version bump

YPares commented:
Hi Sönke,
Github refuses the transfer since I don't have admin rights to
nikki-and-the-robots, so I applied the pull request and added you as a
collaborator to the repository.

Best,

Yves.

2015-09-14 17:31 GMT+02:00 Sönke Hahn notifications@github.com:

> I've relaxed some of the cabal constraints and added a travis file. Since
> I'm listed as the sole maintainer I took the liberty to upload a new
> version to hackage.
> 
> Would you mind transferring the repo to the github organization
> nikki-and-the-robots? Then I would take care of merging this (and
> 
> ## updating the cabal file to reflect the change).
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/YPares/BinaryCommunicator/pull/1
> Commit Summary
> - add travis file
> - tweaked cabal version constraints
> - version bump
> 
> File Changes
> - _A_ .travis.yml
>   https://github.com/YPares/BinaryCommunicator/pull/1/files#diff-0
>   (23)
> - _M_ binary-communicator.cabal
>   https://github.com/YPares/BinaryCommunicator/pull/1/files#diff-1 (7)
> 
> Patch Links:
> - https://github.com/YPares/BinaryCommunicator/pull/1.patch
> - https://github.com/YPares/BinaryCommunicator/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/YPares/BinaryCommunicator/pull/1.

soenkehahn commented:
Thanks @YPares!

What you could try (now since I deleted my fork) is to transfer the repo to my personal account. Then I would transfer it to the organization afterwards.

I wouldn't be insisting on this if it weren't for the travis builds. I'd like to have them for this repo and I've already added the travis file and it builds ok. It seems I can't enable travis for repos where I'm only a collaborator though.

Alternatively you could enable travis builds yourself.
